src/webgl/p5.RendererGL.js

```diff
@@ -631,7 +635,22 @@ p5.RendererGL.prototype._getColorShader = function () {
   return this._defaultColorShader;
 };
 
+p5.RendererGL.prototype.getTexture = function (img) {
+  // efficiency?
+  var f = function(element) {
+    return element.src === img;
+  };
+
+  var tex = this.textures.find(f);
+  if (tex === undefined) {
+    tex = new p5.Texture(this, img);
+    this.textures.push(tex);
+  }
 
+  tex.set(img);
```

Comment:
When logging in the console, I can see that `getTexture()` is called every frame. Since it's a relatively expensive operation to always copy the pixels when we do that, I think we can do something that binds the texture without also copying the pixels.

As a further optimization, once a texture is bound to a given location, the uniform and texture don't need to be rebound unless you want the uniform to point to a different texture or for the texture to change.